<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homePage</class>
 <widget class="QWidget" name="homePage">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>10</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Swis721 Blk BT</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Numerical program</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pictures/appIcon.png</normaloff>../pictures/appIcon.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#homePage{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 191, 127, 255), stop:0.414773 rgba(235, 148, 61, 255), stop:0.875 rgba(248, 203, 255, 255))
}</string>
  </property>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>751</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 30pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255)
</string>
   </property>
   <property name="text">
    <string>Welcome to our numerical application</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nonlinearButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#nonlinearButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	
	font: 20pt &quot;MS Shell Dlg 2&quot;;
}

#nonlinearButton:hover{
	background-color:#584082;
}
</string>
   </property>
   <property name="text">
    <string>Non-linear Equations</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../pictures/nonlinear.png</normaloff>../pictures/nonlinear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="linearButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#linearButton{
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
}

#linearButton:hover{
	background-color:#584082;
}
</string>
   </property>
   <property name="text">
    <string>Linear Equations</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../pictures/linear.png</normaloff>../pictures/linear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#exitButton{
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
}

#exitButton:hover{
	background-color:#584082;
}
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../pictures/exit.jpg</normaloff>../pictures/exit.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
